doc/Changelog.md

```diff
@@ -3,6 +3,12 @@ Changelog {#changelog}
 
 # git master {#master}
 
+* [#55](https://github.com/BlueBrain/Fivox/pull/55)
+  Synapse densities support 
+  - New 'preTarget' and 'postTarget' URI parameters to support synaptic projections
+  - New DensityImageSource that voxelizes by iterating over events rather than voxels
+  - New out-of-core chunking support for event sources, implemented for SynapseLoader
+  - Unify rescaling for non-float volumes; done by ScaleFilter
```

Comment:
Document the API additions in URIHandler (knowing the motivation would also help)